tests_integration/nativeapp/test_version.py

```diff
@@ -12,11 +12,42 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from shlex import split
+from typing import Any, Optional
 
+from yaml import safe_dump, safe_load
+
+from snowflake.cli.api.project.util import (
+    is_valid_unquoted_identifier,
+    to_identifier,
+    unquote_identifier,
+)
 from tests.project.fixtures import *
 from tests_integration.test_utils import contains_row_with, row_from_snowflake_session
 
 
+def set_version_in_app_manifest(manifest_path: Path, version: Any, patch: Any = None):
+    with open(manifest_path, "r") as f:
+        manifest = safe_load(f)
+
+    version_info = manifest.setdefault("version", {})
+    version_info["name"] = version
+    if patch is not None:
+        version_info["patch"] = patch
+    else:
+        version_info.pop("patch", None)
+
+    with open(manifest_path, "w") as f:
+        f.write(safe_dump(manifest))
+
+
+def normalize_identifier(identifier: str) -> str:
+    id_str = str(identifier)
```

Comment:
according to the type this is already a str but I see that the test passes an int, so can we update the type?

Reply:
oops, updated.